types/graphql/language/ast.d.ts

```diff
@@ -424,7 +424,7 @@ export interface FieldDefinitionNode {
     readonly loc?: Location;
     readonly description?: StringValueNode;
     readonly name: NameNode;
-    readonly arguments?: ReadonlyArray<InputValueDefinitionNode>;
+    readonly arguments: ReadonlyArray<InputValueDefinitionNode>;
```

Comment:
It seems that `arguments` is an optional value.
https://github.com/graphql/graphql-js/blob/master/src/language/ast.js#L444

Reply:
Ah! Thanks for the quick response.

It seems like `graphql-js` has a bug in the AST printer then:

https://github.com/graphql/graphql-js/blob/787422956c9554d12d063a41fe35705335ec6290/src/language/printer.js#L139

It fails if `arguments` is indeed undefined. I'll make a PR on their end so it handles it correctly.